calcite/README.md

```diff
@@ -107,4 +107,4 @@ For further details see [Calcite Web TypeFaces](https://esri.github.io/calcite-w
 
 ## Sample app
```

Comment:
This still says sample. Should it be demo for consistency?

Reply:
This has been fixed with commit: https://github.com/Esri/arcgis-maps-sdk-toolkit-qt/pull/660/commits/6c586e2992fc9a8dd2958a9ec3704b7a738cafaa